src/Handler/Search.hs

```diff
@@ -7,6 +7,10 @@ module Handler.Search
 import Import
 import Data.Trie (elems, submap)
 import Data.Version (showVersion)
+import Data.Maybe (isNothing)
```

Comment:
Can you re-export these functions and `MaybeT` from `Import.NoFoundation`? I think they're ubiquitous enough that it's better to do that to help avoid having too many imports here.